parts/k8s/kubernetesagentresourcesvmas.t

```diff
@@ -239,7 +249,7 @@
          "count": "[variables('{{.Name}}Count')]",
          "name": "vmLoopNode"
        },
-       "apiVersion": "2015-05-01-preview",
+       "apiVersion": {{if IsKubernetes}}"2015-05-01-preview"{{else}}"[variables('apiVersionVMDefault')]"{{end}},
```

Comment:
`apiVersionVMDefault` is only set if `IsOpenShift` do we need to be safe here and check that rather than `IsKubernetes`? (same for below)

Reply:
updated